apps/condo/domains/meter/components/CreateMeterReadingsForm.tsx

```diff
@@ -206,6 +203,12 @@ export const CreateMeterReadingsForm = ({ organization, role }) => {
         selectedUnitTypeRef.current = selectedUnitType
     }, [selectedUnitType])
 
+    const { requiredValidator } = useValidations()
+    const { addressValidator } = usePropertyValidations()
+    const validations = {
+        property: [requiredValidator, addressValidator(selectedPropertyId, isMatchSelectedProperty)],
```

Comment:
looks like that here maybe you need to add covering function like so
`() => addressValidator(selectedPropertyId, isMatchSelectedProperty)`

Reply:
addressValidator return fn